src/app/api/v1/bank-account/delete/route.ts

```diff
@@ -0,0 +1,30 @@
+import { checkUserAuthOrThrowError } from "@/app/api/v1/server-actions";
```

Comment:
This comment is for whole file - in general this is great and we need just couple changes here :)

I would love to move this into `api/v1/bank-account/{id}/route.ts` - and as you can see, we already have this there :)) I noticed it right now (I forgot :D that we have it).

So compare your solution and existing solution. And we should have URL for deleting under {id}, when URL should be `DELETE api/v1/bank-account/fe123xxa` rather than `bank-account/delete?fe123xxa`